docs/zh_cn/user_guides/dataset_prepare.md

```diff
@@ -75,6 +75,18 @@ mmdetection3d
 
 ## 数据下载和预处理
 
+### 数据集标注文件列表
+
```

Comment:
Add a description here to illustrate the purpose of prepared `info.pkl` and I suggest putting the list at the end of `dataset_prepare.md`.